Framework/Geometry/inc/MantidGeometry/Objects/Track.h

```diff
@@ -166,9 +167,9 @@ class MANTID_GEOMETRY_DLL Track {
   /// Clear the current set of intersection results
   void clearIntersectionResults();
   /// Returns the starting point
-  const Kernel::V3D &startPoint() const { return m_startPoint; }
+  const Kernel::V3D startPoint() const { return m_line.getOrigin(); }
```

Comment:
Can we keep the `&` here to avoid a copy. We may have to update `Line::getOrigin()` to return `const V3D &` too? The same thing for direction too.